src/main/resources/liquibase/crop_changelog/14_2_0.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="gelli" id="v14.2.0-1">
+        <preConditions onFail="MARK_RAN">
+            <and>
+                <sqlCheck expectedResult="1">
+                    SELECT CASE WHEN (SELECT COUNT(*) FROM germplsm WHERE methn = 0) > 0 THEN 1 ELSE 0 END;
+                </sqlCheck>
+                <sqlCheck expectedResult="2">
+                    SELECT COUNT(*) FROM methods WHERE mid IN (1,31);
+                </sqlCheck>
+            </and>
+        </preConditions>
+        <comment>Update records with germplasm.methn = 0 or germplasm.methn not existing to methods </comment>
+        <sql dbms="mysql" >
+            UPDATE germplsm SET methn =
```

Comment:
For any changeset, we should only update the records we need to. Hence, we shouldn't be updating the whole `germplsm` table but only the records that we needed to (i.e. methn = 0 or methn doesn't exist in method table).

As such, we can simplify the script as:
UPDATE germplsm SET 
  methn = CASE WHEN gnpgs <= 0 THEN 31
	         ELSE 1
		END
WHERE methn = 0 OR NOT EXISTS (SELECT 1 FROM methods WHERE mid = methn);

`